internal/signalio/helpers.go

```diff
@@ -36,11 +36,9 @@ func marshalToMap(signals []signal.Set, extra ...Field) (map[string]string, erro
 	for _, s := range signals {
 		// Get all of the signal data from the set and serialize it.
 		for k, v := range signal.SetAsMap(s, true) {
-			if s, err := marshalValue(v); err != nil {
-				return nil, fmt.Errorf("failed to write field %s: %w", k, err)
-			} else {
-				values[k] = s
-			}
+			s, _ := marshalValue(v) //nolint:nolintlint
```

Comment:
`marshalValue()` should probably be in this file.

Reply:
I have added `marshalValue()` into this file because the only usages of `marshalValue()` is in `helpers.go` and `helpers_test.go`.